packages/app-desktop/gui/ResizableLayout/utils/useLayoutItemSizes.test.ts

```diff
@@ -20,6 +20,86 @@ describe('useLayoutItemSizes', () => {
 		expect(layout.isRoot).toBe(true);
 	});
 
+	test('should strecth the last visible child item if all siblings have fixed size and the last child is not visible', () => {
+		const layout: LayoutItem = validateLayout({
+			key: 'root',
+			width: 200,
+			height: 100,
+			direction: LayoutItemDirection.Row,
+			children: [
+				{ key: 'col1', width: 50 },
+				{ key: 'col2', width: 50 },
+				{ key: 'col3', width: 70, visible: false },
+			],
+		});
+
+		expect(layout.children).toEqual([
+			{
+				key: 'col1',
+				width: 50,
+				resizableRight: true,
+				resizableBottom: false,
+				direction: 'column',
+				visible: true,
+			},
+			{
+				key: 'col2',
+				resizableRight: false,
+				resizableBottom: false,
+				direction: 'column',
+				visible: true,
+			},
+			{
+				key: 'col3',
+				width: 70,
+				resizableRight: true,
+				resizableBottom: false,
+				direction: 'column',
+				visible: false,
+			},
+		]);
```

Comment:
Yesterday I missed this somehow, but it seems I have to test all properties of all children that `validateLayout` adds. Do you think this approach is okay?

Reply:
Yes that's a good point, you should indeed just test the relevant properties, which in this case are "width" and "visible" I guess. Usually I'd do this like so:

```typescript
const col1 = layout.children.find(c => c.key === 'col1');
expect(col1.width ...
expect(col1.visibible...
// etc.
```